app/views/credits/new.html.erb

```diff
@@ -137,7 +137,7 @@
     var stripe = Stripe('<%= ApplicationConfig["STRIPE_PUBLISHABLE_KEY"] %>');
     var elements = stripe.elements();
 
-    <% if current_user.config_theme == "night_theme" %>
+    <% if current_user.config_theme == "night_theme" || current_user.config_theme == "ten_x_hacker_theme" %>
```

Comment:
maybe this should be a "if current_user.dark_themes?", to avoid having to change this file for each dark theme we add